assets/src/sass/0-settings/_colors.scss

```diff
@@ -3,10 +3,10 @@
 		Colors Variables
 -----------------------------------*/
 
-$black: #000000;
-$white: #ffffff;
+$black: #000;
+$white: #fff;
```

Comment:
Better to create objects of colors, that will help to generate Gutenberg classes.

Reference from rtCamp repo:

**Object of Colors:**
https://github.com/rtCamp/rtcamp.com/blob/master/themes/rtcamp/assets/src/sass/0-settings/_colors.scss#L27-L40

**Color Function:** - This is to retrieve hex color code from the color key.
https://github.com/rtCamp/rtcamp.com/blob/master/themes/rtcamp/assets/src/sass/0-settings/_functions.scss#L6-L20

**Generate Color Classes:** (Few classes from below ref. are theme-specific so please ignore those, only add background and color classes)
https://github.com/rtCamp/rtcamp.com/blob/master/themes/rtcamp/assets/src/sass/2-generic/_color-classes.scss

Reply:
are there any other mixins/functions you may have created on any other client project that can be added here?

Reply:
You can include the above one for now.